maven/src/test/resources/pom-processor/repositories-expression-in-a-profile/test-pom-1.0.pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a good test to check, but the one I mentioned above might involve redeclaring a property value for repo.one.url and using it in all of the repository declarations. For repositories in profiles where the value is redeclared, it should resolve to a value from the profile. For repositories declared in the main pom.xml, it should use the value from the main pom.xml.

Another interesting twist we should check is to leave the property declaration out of the second profile, and use the same expression for the repository url. This should resolve to the value from the main pom.xml.